go.mod

```diff
@@ -0,0 +1,68 @@
+module github.com/liatrio/lead-terraform
+
+go 1.17
+
+require (
+	github.com/aws/aws-sdk-go v1.41.11
+	github.com/gruntwork-io/terratest v0.38.2
+	github.com/stretchr/testify v1.7.0
+	k8s.io/api v0.20.6 // indirect
+	k8s.io/apimachinery v0.20.6
+)
+
+require (
+	cloud.google.com/go v0.83.0 // indirect
```

Comment:
Go 1.17 moves [transitive deps into the go.mod](https://golang.org/doc/go1.17#go-command)